lib/heroku/command/pg_backups.rb

```diff
@@ -397,7 +397,9 @@ def unschedule_backups
     attachment = generate_resolver.resolve(db, "DATABASE_URL")
 
     schedule = hpg_client(attachment).schedules.find do |s|
-      attachment.name =~ /#{s[:name]}/
+      # attachment.name is HEROKU_POSTGRESQL_COLOR
+      # s[:name] is HEROKU_POSTGRESQL_COLOR_URL
+      /#{attachment.name}/ =~ s[:name]
```

Comment:
Actually, maybe let's include both checks with `||`--that will make sure we handle older schedules as well.